packages/ui/src/components/Toggle/index.tsx

```diff
@@ -114,30 +115,6 @@ const StyledLabel = styled.label<{
   }
 `
 
-const StyledLabelContent = styled.span<{
-  labelPosition: 'left' | 'right'
-}>`
-  ${({ theme, labelPosition }) =>
-    labelPosition === 'left'
-      ? `margin-right: ${theme.space['1']}`
-      : `margin-left: ${theme.space['1']}`}
-`
-
-const LabelContent = ({
-  label,
-  labelPosition,
-}: Required<Pick<ToggleProps, 'label' | 'labelPosition'>>) => {
-  if (typeof label === 'string') {
```

Comment:
Why lose this handling of string vs Component ?

Reply:
Because we always want to have a gap between the Toggle itself and the label.

Reply:
This can still be the case while keeping the handling of string vs component no ?

Reply:
@matthprost do we want to always have a gap between the toggle and label? If yes, handling string differently from components becomes useless

Reply:
I don't understand, either way we are going to have a gap between them


```tsx
<Toggle label="sample label" />
```

will render a proper span with a gap of 8 inside the Toggle

```tsx
<Toggle label={<Text as="span" variant="body">sample label</Text>} />
```

Will render the same gap with a custom component 


I fail to understand why the LabelContent is related to the gap ? :)

Reply:
Resolving since as explained we won't care about making a difference between string vs components, as we always want a gap between the toggle and the label.